### PR TITLE
bump conmon to 2.0.8

### PR DESCRIPTION
**Pull request by haircommander**
As well as implement versioning scheme for circle images, where we push to \<branch>-\<sem-version>, as opposed to \<sem-version>.

Signed-off-by: Peter Hunt <pehunt@redhat.com>

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM once kata-jenkins likes it.

File: go.sum [hunk 1]
```diff
@@ -209,6 +209,10 @@ github.com/containers/conmon v2.0.2+incompatible h1:h2HCdd/EBpwFn7RT82Y2GyXnVUHW
 github.com/containers/conmon v2.0.2+incompatible/go.mod h1:hgwZ2mtuDrppv78a/cOBNiCm6O0UMWGx1mu7P00nu5I=
 github.com/containers/conmon v2.0.3+incompatible h1:jxH9Nm1owQ8spsmISHJGiUChqTKrw9BUuEhdR8196bE=
 github.com/containers/conmon v2.0.3+incompatible/go.mod h1:hgwZ2mtuDrppv78a/cOBNiCm6O0UMWGx1mu7P00nu5I=
+github.com/containers/conmon v2.0.6+incompatible h1:FCnBv2AnxClfvpRlg5moKYPi2pBFVLRUfsoaql2z59s=
```
Comment (mrunalp) [inline go.sum]: why are these piling up? :/

Comment (haircommander) [reply go.sum]: https://github.com/renovatebot/renovate/issues/2594#issuecomment-426507432
kind of gives an answer

**Comment by haircommander** [comment]
/hold
```
# (in test file ./ctr.bats, line 1163)
#   `[[ "$output" =~ "OOMKilled" ]]' failed
# 0
```

**Comment by mrunalp** [comment]
@giuseppe fyi

**Pull request merged**